deployment/ccip/changeset/token_pools.go

```diff
@@ -17,13 +17,14 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/evm/utils"
 )
 
-var currentTokenPoolVersion semver.Version = deployment.Version1_5_1
+var CurrentTokenPoolVersion semver.Version = deployment.Version1_5_1
 
 var tokenPoolTypes map[deployment.ContractType]struct{} = map[deployment.ContractType]struct{}{
 	BurnMintTokenPool:         struct{}{},
 	BurnWithFromMintTokenPool: struct{}{},
 	BurnFromMintTokenPool:     struct{}{},
 	LockReleaseTokenPool:      struct{}{},
+	USDCTokenPool:             struct{}{},
```

Comment:
need to have hybridUSDC here as well?

Reply:
https://github.com/smartcontractkit/chainlink/pull/16062/commits/8d1f3368c00ba86eb007e554b4aa949a31a9fb80